pallets/staking/src/lib.rs

```diff
@@ -1825,7 +1825,7 @@ pub mod pallet {
             stash: T::AccountId,
             num_slashing_spans: u32,
         ) -> DispatchResult {
-            let at_minimum = T::Currency::total_balance(&stash) == T::Currency::minimum_balance();
+            let at_minimum = T::Currency::total_balance(&stash) == <ExistentialDeposit<T>>::get();
```

Comment:
no need to change, see: https://github.com/paritytech/substrate/blob/master/frame/staking/src/pallet/mod.rs#L1518